pkg/csi/service/vanilla/controller_test.go

```diff
@@ -637,3 +647,27 @@ func TestCompleteControllerFlow(t *testing.T) {
 		t.Fatalf("Volume should not exist after deletion with ID: %s", volID)
 	}
 }
+
+func GetContainerOrchestratorInterface(orchestratorType int) (commonco.COCommonInterface, error) {
```

Comment:
`func GetContainerOrchestratorInterface(orchestratorType int) (commonco.COCommonInterface, error)`
and `func (c *FakeK8SOrchestrator) IsFSSEnabled(ctx context.Context, featureName string) bool` are present in pkg/csi/service/vanilla/controller_test.go and pkg/syncer/syncer_test.go file

Can we avoid code duplication?

Reply:
These method have to be mocked in both the test files. I don't think we can have a util function common to all tests.

Reply:
We can move the common unit test code to `pkg/common/common_test.go`

Reply:
DOne